robyn/cli.py

```diff
@@ -104,6 +105,15 @@ def start_app_normally(config: Config):
 
 def run():
     config = Config()
+
+    if not config.file_path:
+        config.file_path = f"{os.getcwd()}/{__name__}"
```

Comment:
is it needed?

Reply:
yes, otherwise the file path will be `None`, as we discussed in the previous meeting.